bindings/ruby/lib/typelib/clang.rb

```diff
@@ -29,6 +29,13 @@ def self.load(registry, file, kind, options)
             
             include_dirs = options[:include_paths] || []
             include_path = include_dirs.map { |d| "-I#{d}" }
+            defines = options[:define] || []
+            #add -D to the defines, if its missing
+            defines.each do |define|
+                if not define.start_with? '-D'
```

Comment:
I don't really understand what you are implying? Do you want me to forward the defines without -D to the importer and add the -D in the importer code?